docs/content/en/tasks/connect/ingress/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: "Configure Traefik ingress"
 linkTitle: "Traefik ingress"
+toc_hide: true
```

Comment:
@johnsmartco and I discussed what should happen with the current `tasks/connect/ingress` sections that are still available, even though the menu itself is hidden -- while most of the content there still applies, since it's mostly "how do I configure kind to allow ingress" sort of stuff, it's not going to be perfect in the new world.  John will file a separate issue to clean that aspect of things up.

Reply:
https://github.com/k8ssandra/k8ssandra/issues/1335